### PR TITLE
Update regions.txt

### PR DESCRIPTION
**Pull request by IohannesIohannium**
Adds color (as in pull request https://github.com/ParadoxGameConverters/Vic2ToHoI4/pull/1414), writes out the region groupings at the head. Still not written them down entirely.

### CODE REVIEW

File: Vic2ToHoI4/Data_Files/configurables/regions.txt [hunk 1]
```diff
@@ -1254,27 +1317,31 @@ indochina = {
...
+#malaysia = {
+#	name = "malaya"
+#	adjective = "malaya_adj"
+#	freed_name = "malaysia"
+#	freed_adjective = "malaysia_adj"
+#	level = autonomy_colony
+#	color = { 106 226 174 }
+#	provinces = {
+#	}
+#}
```
Comment (Idhrendur) [inline Vic2ToHoI4/Data_Files/configurables/regions.txt]: Just go ahead and delete this entirely.

File: Vic2ToHoI4/Data_Files/configurables/regions.txt [hunk 2]
```diff
@@ -301,7 +340,7 @@ baltic = {
 	freed_name = "baltic"
 	freed_adjective = "baltic_adj"
 	level = autonomy_colony
-	blocked = { europe }
```
Comment (cetvrtak) [inline Vic2ToHoI4/Data_Files/configurables/regions.txt:304]: Restore this

File: Vic2ToHoI4/Data_Files/configurables/regions.txt [hunk 3]
```diff
@@ -592,7 +637,7 @@ romania = {
 	freed_name = "romania"
 	freed_adjective = "romania_adj"
 	level = autonomy_integrated_puppet
-	blocked = { europe }
```
Comment (cetvrtak) [inline Vic2ToHoI4/Data_Files/configurables/regions.txt:595]: And this

File: Vic2ToHoI4/Data_Files/configurables/regions.txt [hunk 4]
```diff
@@ -766,7 +816,7 @@ turkey = {
 	freed_name = "turkey"
 	freed_adjective = "turkey_adj"
 	level = autonomy_integrated_puppet
-	blocked = { europe }
```
Comment (cetvrtak) [inline Vic2ToHoI4/Data_Files/configurables/regions.txt:769]: ...probably this too, until we figure out the exact position of Turkey

File: Vic2ToHoI4/Data_Files/configurables/regions.txt [hunk 5]
```diff
@@ -1009,7 +1065,7 @@ transcaucasia = {
 	freed_name = "transcaucasia"
 	freed_adjective = "transcaucasia_adj"
 	level = autonomy_integrated_puppet
-	blocked = { europe }
```
Comment (cetvrtak) [inline Vic2ToHoI4/Data_Files/configurables/regions.txt:1012]: Same goes as for Turkey

File: Vic2ToHoI4/Data_Files/configurables/regions.txt (hunk 2 above)
Comment (IohannesIohannium) [reply Vic2ToHoI4/Data_Files/configurables/regions.txt:304]: Deleted by accident, my bad

**Review by cetvrtak — APPROVED** [review]

**Pull request merged**